executor/runtime/container.go

```diff
@@ -9,13 +9,17 @@ import (
 )
 
 const (
-	appNameLabelKey        = "com.netflix.titus.appName"
-	cpuLabelKey            = "com.netflix.titus.cpu"
-	memLabelKey            = "com.netflix.titus.mem"
-	diskLabelKey           = "com.netflix.titus.disk"
-	networkLabelKey        = "com.netflix.titus.network"
-	workloadTypeLabelKey   = "com.netflix.titus.workload.type"
-	titusTaskInstanceIDKey = "TITUS_TASK_INSTANCE_ID"
+	appNameLabelKey          = "com.netflix.titus.appName"
+	cpuLabelKey              = "com.netflix.titus.cpu"
+	memLabelKey              = "com.netflix.titus.mem"
+	diskLabelKey             = "com.netflix.titus.disk"
+	networkLabelKey          = "com.netflix.titus.network"
+	workloadTypeLabelKey     = "com.netflix.titus.workload.type"
+	ownerEmailLabelKey       = "com.netflix.titus.owner.email"
+	ownerEmailPassThroughKey = "titus.agent.ownerEmail"
```

Comment:
why not `com.netflix.titus`?

Reply:
This is the labeling scheme from the control plane as suggested by @corindwyer.
https://github.com/Netflix/titus-control-plane/blob/master/titus-server-master/src/main/java/com/netflix/titus/master/mesos/DefaultV3TaskInfoFactory.java#L66